### ISSUE TITLE
Ammonite is unable to run from SBT 1.0.4

### ISSUE THREAD

**Issue by Rorick** [lihaoyi/Ammonite#738]
Ammonite fails on start with exception:
```
[info] Running amm 
Loading...
Compiling (synthetic)/ammonite/predef/interpBridge.sc
java.lang.NoClassDefFoundError: scala/collection/Iterator
  java.lang.Class.getDeclaredMethods0(Native Method)
  java.lang.Class.privateGetDeclaredMethods(Class.java:2701)
  java.lang.Class.getDeclaredMethod(Class.java:2128)
  ammonite.runtime.Evaluator$$anon$1.$anonfun$evalMain$1(Evaluator.scala:92)
  ammonite.util.Util$.withContextClassloader(Util.scala:16)
  ammonite.runtime.Evaluator$$anon$1.evalMain(Evaluator.scala:92)
  ammonite.runtime.Evaluator$$anon$1.$anonfun$processScriptBlock$1(Evaluator.scala:126)
  ammonite.util.Catching.map(Res.scala:116)
  ammonite.runtime.Evaluator$$anon$1.processScriptBlock(Evaluator.scala:124)
  ammonite.interp.Interpreter.$anonfun$processSingleBlock$4(Interpreter.scala:286)
  ammonite.util.Res$Success.flatMap(Res.scala:61)
  ammonite.interp.Interpreter.$anonfun$processSingleBlock$3(Interpreter.scala:279)
  ammonite.util.Res$Success.flatMap(Res.scala:61)
  ammonite.interp.Interpreter.$anonfun$processSingleBlock$1(Interpreter.scala:276)
  ammonite.util.Catching.flatMap(Res.scala:114)
  ammonite.interp.Interpreter.processSingleBlock(Interpreter.scala:275)
  ammonite.interp.Interpreter.$anonfun$processModule$8(Interpreter.scala:344)
  ammonite.interp.Interpreter.$anonfun$processAllScriptBlocks$3(Interpreter.scala:470)
  ammonite.util.Res$Success.flatMap(Res.scala:61)
  ammonite.interp.Interpreter.compileRunBlock$1(Interpreter.scala:458)
  ammonite.interp.Interpreter.$anonfun$processAllScriptBlocks$9(Interpreter.scala:495)
  scala.Option$WithFilter.map(Option.scala:146)
  ammonite.interp.Interpreter.loop$1(Interpreter.scala:482)
  ammonite.interp.Interpreter.processAllScriptBlocks(Interpreter.scala:546)
  ammonite.interp.Interpreter.$anonfun$processModule$6(Interpreter.scala:346)
  ammonite.util.Catching.flatMap(Res.scala:114)
  ammonite.interp.Interpreter.$anonfun$processModule$5(Interpreter.scala:337)
  ammonite.util.Res$Success.flatMap(Res.scala:61)
  ammonite.interp.Interpreter.processModule(Interpreter.scala:329)
  ammonite.interp.Interpreter.$anonfun$initializePredef$1(Interpreter.scala:104)
  ammonite.interp.Interpreter.$anonfun$initializePredef$1$adapted(Interpreter.scala:104)
  ammonite.interp.PredefInitialization$.$anonfun$apply$7(PredefInitialization.scala:65)
  ammonite.util.Res$.$anonfun$fold$1(Res.scala:31)
  scala.collection.LinearSeqOptimized.foldLeft(LinearSeqOptimized.scala:122)
  scala.collection.LinearSeqOptimized.foldLeft$(LinearSeqOptimized.scala:118)
  scala.collection.immutable.List.foldLeft(List.scala:86)
  ammonite.util.Res$.fold(Res.scala:29)
  ammonite.interp.PredefInitialization$.apply(PredefInitialization.scala:53)
  ammonite.interp.Interpreter.initializePredef(Interpreter.scala:106)
  ammonite.repl.Repl.initializePredef(Repl.scala:107)
  ammonite.Main.run(Main.scala:181)
  ammonite.MainRunner.$anonfun$runRepl$1(Main.scala:363)
  ammonite.MainRunner.watchLoop(Main.scala:344)
  ammonite.MainRunner.runRepl(Main.scala:363)
  ammonite.Main$.main0(Main.scala:280)
  ammonite.Main$.main(Main.scala:244)
  amm$.delayedEndpoint$amm$1(amm.scala:1)
  amm$delayedInit$body.apply(amm.scala:1)
  scala.Function0.apply$mcV$sp(Function0.scala:34)
  scala.Function0.apply$mcV$sp$(Function0.scala:34)
  scala.runtime.AbstractFunction0.apply$mcV$sp(AbstractFunction0.scala:12)
  scala.App.$anonfun$main$1$adapted(App.scala:76)
  scala.collection.immutable.List.foreach(List.scala:389)
  scala.App.main(App.scala:76)
  scala.App.main$(App.scala:74)
  amm$.main(amm.scala:1)
  amm.main(amm.scala)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
  java.lang.reflect.Method.invoke(Method.java:498)
  sbt.Run.invokeMain(Run.scala:89)
  sbt.Run.run0(Run.scala:83)
  sbt.Run.execute$1(Run.scala:61)
  sbt.Run.$anonfun$run$4(Run.scala:73)
  scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:12)
  sbt.util.InterfaceUtil$$anon$1.get(InterfaceUtil.scala:10)
  sbt.TrapExit$App.run(TrapExit.scala:252)
  java.lang.Thread.run(Thread.java:745)
java.lang.ClassNotFoundException: scala.collection.Iterator
  java.net.URLClassLoader.findClass(URLClassLoader.java:381)
  ammonite.runtime.SpecialClassLoader.findClass(ClassLoaders.scala:198)
  java.lang.ClassLoader.loadClass(ClassLoader.java:424)
  java.lang.ClassLoader.loadClass(ClassLoader.java:357)
  java.lang.Class.getDeclaredMethods0(Native Method)
  java.lang.Class.privateGetDeclaredMethods(Class.java:2701)
  java.lang.Class.getDeclaredMethod(Class.java:2128)
  ammonite.runtime.Evaluator$$anon$1.$anonfun$evalMain$1(Evaluator.scala:92)
  ammonite.util.Util$.withContextClassloader(Util.scala:16)
  ammonite.runtime.Evaluator$$anon$1.evalMain(Evaluator.scala:92)
  ammonite.runtime.Evaluator$$anon$1.$anonfun$processScriptBlock$1(Evaluator.scala:126)
  ammonite.util.Catching.map(Res.scala:116)
  ammonite.runtime.Evaluator$$anon$1.processScriptBlock(Evaluator.scala:124)
  ammonite.interp.Interpreter.$anonfun$processSingleBlock$4(Interpreter.scala:286)
  ammonite.util.Res$Success.flatMap(Res.scala:61)
  ammonite.interp.Interpreter.$anonfun$processSingleBlock$3(Interpreter.scala:279)
  ammonite.util.Res$Success.flatMap(Res.scala:61)
  ammonite.interp.Interpreter.$anonfun$processSingleBlock$1(Interpreter.scala:276)
  ammonite.util.Catching.flatMap(Res.scala:114)
  ammonite.interp.Interpreter.processSingleBlock(Interpreter.scala:275)
  ammonite.interp.Interpreter.$anonfun$processModule$8(Interpreter.scala:344)
  ammonite.interp.Interpreter.$anonfun$processAllScriptBlocks$3(Interpreter.scala:470)
  ammonite.util.Res$Success.flatMap(Res.scala:61)
  ammonite.interp.Interpreter.compileRunBlock$1(Interpreter.scala:458)
  ammonite.interp.Interpreter.$anonfun$processAllScriptBlocks$9(Interpreter.scala:495)
  scala.Option$WithFilter.map(Option.scala:146)
  ammonite.interp.Interpreter.loop$1(Interpreter.scala:482)
  ammonite.interp.Interpreter.processAllScriptBlocks(Interpreter.scala:546)
  ammonite.interp.Interpreter.$anonfun$processModule$6(Interpreter.scala:346)
  ammonite.util.Catching.flatMap(Res.scala:114)
  ammonite.interp.Interpreter.$anonfun$processModule$5(Interpreter.scala:337)
  ammonite.util.Res$Success.flatMap(Res.scala:61)
  ammonite.interp.Interpreter.processModule(Interpreter.scala:329)
  ammonite.interp.Interpreter.$anonfun$initializePredef$1(Interpreter.scala:104)
  ammonite.interp.Interpreter.$anonfun$initializePredef$1$adapted(Interpreter.scala:104)
  ammonite.interp.PredefInitialization$.$anonfun$apply$7(PredefInitialization.scala:65)
  ammonite.util.Res$.$anonfun$fold$1(Res.scala:31)
  scala.collection.LinearSeqOptimized.foldLeft(LinearSeqOptimized.scala:122)
  scala.collection.LinearSeqOptimized.foldLeft$(LinearSeqOptimized.scala:118)
  scala.collection.immutable.List.foldLeft(List.scala:86)
  ammonite.util.Res$.fold(Res.scala:29)
  ammonite.interp.PredefInitialization$.apply(PredefInitialization.scala:53)
  ammonite.interp.Interpreter.initializePredef(Interpreter.scala:106)
  ammonite.repl.Repl.initializePredef(Repl.scala:107)
  ammonite.Main.run(Main.scala:181)
  ammonite.MainRunner.$anonfun$runRepl$1(Main.scala:363)
  ammonite.MainRunner.watchLoop(Main.scala:344)
  ammonite.MainRunner.runRepl(Main.scala:363)
  ammonite.Main$.main0(Main.scala:280)
  ammonite.Main$.main(Main.scala:244)
  amm$.delayedEndpoint$amm$1(amm.scala:1)
  amm$delayedInit$body.apply(amm.scala:1)
  scala.Function0.apply$mcV$sp(Function0.scala:34)
  scala.Function0.apply$mcV$sp$(Function0.scala:34)
  scala.runtime.AbstractFunction0.apply$mcV$sp(AbstractFunction0.scala:12)
  scala.App.$anonfun$main$1$adapted(App.scala:76)
  scala.collection.immutable.List.foreach(List.scala:389)
  scala.App.main(App.scala:76)
  scala.App.main$(App.scala:74)
  amm$.main(amm.scala:1)
  amm.main(amm.scala)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
  java.lang.reflect.Method.invoke(Method.java:498)
  sbt.Run.invokeMain(Run.scala:89)
  sbt.Run.run0(Run.scala:83)
  sbt.Run.execute$1(Run.scala:61)
  sbt.Run.$anonfun$run$4(Run.scala:73)
  scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:12)
  sbt.util.InterfaceUtil$$anon$1.get(InterfaceUtil.scala:10)
  sbt.TrapExit$App.run(TrapExit.scala:252)
  java.lang.Thread.run(Thread.java:745)

Exception: sbt.TrapExitSecurityException thrown from the UncaughtExceptionHandler in thread "run-main-0"
```
SBT 0.13.16 runs it fine.

**Comment by lev112** [comment]
I had the same problem, but it was fixed after upgrading to sbt 1.1.0

**Comment by lihaoyi** [comment]
Closing this if fixed in 1.1.0
